### ISSUE TITLE
[a11y] 9.2.4.3 - Logical order on keyboad operation

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5392]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
- **Pages 1 and 2:**
**a)** When clicking the actions "Entferne Ablaufdatum" or "Entferne Passwort" the respective field will be deleted and the buttons removed but the keyboard focus disapears. It is better to move the focus back to the respective input fields.
**b)** When closing the area "Bearbeite öffentlichen Link" with the button "Abbrechen" the keyboard focus does not move back to the button "Bearbeite  öffentlichen Link".
**Note:** the functionality "Freigabe bearbeiten" could not be tested as it was not working - the area didn’t
open.
- **The date picker** dialogue (at least with Firefox) disrupts the correct order when navigation with the keyboard.

## Status: partly fulfilled
- **Page 1 (Alle Dateien)**
When closing the modal dialogue "Neuen Ordner erstellen" with the button "Abbrechen" the keyboard focus does not move back to the button "Neu" but is moved to the top of the page.
See general notes
- **Page 2 (Mit anderen geteilten Dateien)**
See general notes

## Status: fulfilled
- **Page 3 (Gelöschte Dateien)**
- **Page 4 (Konto)**

**Comment by pascalwengerter** [comment]
Date picker was expected to fail in the audit, not sure if we should (finally) address it for the re-audit?

**Comment by kulmann** [comment]
Yes, we need to do that

**Comment by pascalwengerter** [comment]
General notes 1a) ""Entferne Passwort"" is handled in f7a67b9dbe26e82d4990d4b98c7a5808d9ea9d25

**Comment by pascalwengerter** [comment]
Status: partly fulfilled, Page 1, is handled in 8a328e36ebf48fa723fbc04c4fa7ec31e510b95d

**Comment by pascalwengerter** [comment]
General notes 1b) Focus management when "Bearbeiten öffentlicher Link" handled in 5c3ae317b52f4a55b42258f96a89836515208d64

**Comment by pascalwengerter** [comment]
Closed as superseded by #10733
